docs/source/docker/advanced/sequence_selection.rst

```diff
@@ -0,0 +1,80 @@
+Sequence Selection
+==================
+
+Sequence selection allows users to select sequences of a video instead of single frames.
+The key concept is the parameter `selected_sequence_length`. If its value is one (default),
+the docker selects single frames. If it is larger than one, each video is split into 
+sequences of that length and the frame representations are aggregated into a sequence
+representation. The sampling then happens on these sequence representations.
+
+.. note:: Sequence selection works on videos or on folders of alphabetically sorted
+    frames.
+
+
+- | If you're interested in how the sequence selection works, go to
+  | --> `How It Works`_
+
+- | To see how you can use the sequence selection, check out
+  | --> `Usage`_
+
+
+How It Works
+-------------
+Sequence selection consists of the following steps:
+
+- Each input video is split into sequences of length `selected_sequence_length`.
+- Next, the embeddings of all frames in a sequence are aggregated (averaged).
+- The Lightly sampling algorithm selects relevant sequences.
+- Finally, the indices of the selected sequence frames are reconstructed.
+- The report is generated and (if requested) the selected frames are saved.
+  
+
+Usage
+-----------
+
+To select sequences of length `X` simply add the argument `selected_sequence_length=X`
```

Comment:
Can we highlight somehow the parameters we can use more? E.g. make them bold or separate them? And also explain what values we can use. Any number from 1 to N? Or is there any "rule" we should follow?